### ISSUE TITLE
Sorry, I can't create a new issue in "NaughtyDogLocalizationTool" so I have to ask here.

### ISSUE THREAD

**Issue by dinhhoangthang** [rm-NoobInCoding/UnPSARC#13]
- I used the latest version 1.1, the errors still appear, I use for the game "UNCHARTED Legacy of Thieves Collection" purchased at gog.com
- After packaging the file and copying it to the game folder, the game cannot start.
- I tested the packaged file using this tool. When decoding the file again, this error message will appear.

![Screenshot 2024-05-25 070127](https://github.com/rm-NoobInCoding/UnPSARC/assets/163383208/9475091a-7ddb-4a0c-b3e1-4a597fc72e97)

**Comment by dinhhoangthang** [comment]
I would like to send the original text2.psarc file for you to check. https://drive.google.com/file/d/1Vz6x2ukV3ZnylyTXRJd3_D_r3xgQi3_j/view?usp=sharing

**Comment by dinhhoangthang** [comment]
Thank you so much for creating these great tools. so we can translate the game into our country's language.

**Comment by dinhhoangthang** [comment]
Thank you so much for creating these great tools.

**Comment by rm-NoobInCoding** [comment]
Its not a bug. 
You must unpack psarc file (to get language files) then use localization tool for them.

**Comment by dinhhoangthang** [comment]
Sorry for the misunderstanding, I sent the above file for you to check to see if there is a difference between the steam and gog platforms.

Because I unpacked the text2.psarc file to get the eng.common and eng.subtitles files and used the NaughtyDogLocalizationTool v1.1 tool to decode and encode it. After encrypting the file and moving it to the game folder, the game cannot start. I tried decoding the file again and it appeared like on the image.
